mp/src/game/client/fortress/hintitembase.h

```diff
@@ -23,13 +23,13 @@ namespace vgui
 }
 
 #define DECLARE_HINTITEMFACTORY( className ) \
-	CHintItemBase *Create_##className##( vgui::Panel *parent, const char *panelName ) \
```

Comment:
A note to the changes here: The `##` operator is for creating a *singular* token. In this instance, for example, pasting `Create_` with `className` with `(` will never create a valid *singular* token. Here two tokens are to be created: `Create_##className` `(`.
gcc is more strict than MSVC here in that it rejects the operation if pasting fails.